### PR TITLE
enable citation import from source code

### PR DESCRIPTION
**Pull request by katrinleinweber**
Hello!

As discussed in [#3480](https://github.com/CellProfiler/CellProfiler/issues/3480). I think I got all the BibTeX snippets of the papers on [cellprofiler.org/citations](http://cellprofiler.org/citations/). Where the publisher didn't offer BibTeX, I imported them to [Zotero](http://zotero.org/) & exported with [Better BibTeX](https://retorque.re/zotero-better-bibtex/).

Cheers :-)

### CODE REVIEW

**Comment by katrinleinweber** [comment]
Ah! Possible mistake: The snippet for CellProfiler Analyst should go into a dedicated `CITATION` file in that repo. Shall I change that?

File: CITATION [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+abstract="Biologists can now prepare and image thousands of samples per day using automation, enabling chemical screens and functional genomics (for example, using RNA interference). Here we describe the first free, open-source system designed for flexible, high-throughput cell image analysis, CellProfiler. CellProfiler can address a variety of biological questions quantitatively, including standard assays (for example, cell count, size, per-cell protein levels) and complex morphological assays (for example, cell/organelle shape or subcellular patterns of DNA or protein staining).",
+issn="1474-760X",
+doi="10.1186/gb-2006-7-10-r100",
+url="https://doi.org/10.1186/gb-2006-7-10-r100"
+}
+
+or
+
+@article{doi:10.1093/bioinformatics/btr095,
+author = {Kamentsky, Lee and Jones, Thouis R. and Fraser, Adam and Bray, Mark-Anthony and Logan, David J. and Madden, Katherine L. and Ljosa, Vebjorn and Rueden, Curtis and Eliceiri, Kevin W. and Carpenter, Anne E.},
```
Comment (mcquin) [inline CITATION]: Please indent all fields between `{...}`s by one tab/two spaces.

File: CITATION [hunk 2]
```diff
@@ -0,0 +1,126 @@
+When using CellProfiler, please cite one of the following papers.
+See also: http://cellprofiler.org/citations/
+
+
+CellProfiler for cells:
+
+@Article{Carpenter2006,
+author="Carpenter, Anne E.
```
Comment (mcquin) [inline CITATION]: Please indent all fields between `{...}`s by one tab/two spaces.

Comment (katrinleinweber) [reply CITATION]: Do you know a linter? I'm using Atom right now, and the auto-format doesn't work on those snippets. Copying verbatim did seem a good idea at the time, but which formatting standard should it be made to adhere to now?

Comment (mcquin) [reply CITATION]: I don't know of a linter specific to this formatting. I'll fix the formatting before the merge.

Comment (katrinleinweber) [reply CITATION]: OK, thank you! If I find a BibTeX linter, I'll let you know.

**Pull request merged**